ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -150,7 +150,7 @@ function extract_oc(){
     echo -e "Extracting oc\n"
     local retry=5 tmp_oc="/tmp/client-2"
     mkdir -p ${tmp_oc}
-    while ! (oc adm release extract -a "${CLUSTER_PROFILE_DIR}/pull-secret" --command=oc --to=${tmp_oc} ${TARGET});
+    while ! (/cli/oc adm release extract -a "${CLUSTER_PROFILE_DIR}/pull-secret" --command=oc --to=${tmp_oc} ${TARGET});
```

Comment:
The error is `error: image did not contain usr/bin/oc`,  not the same as `oc: command not found...`

I don't think the change fix the issue.

Reply:
oh, do you have any idea why we ran into the error?

Reply:
Looks like the command `oc adm release extract` try to extract oc binary from location `usr/bin/oc` in `TARGET`, but TARGET does not have oc binary in `usr/bin/oc`

Reply:
Thanks!

Reply:
I logged in to the upgrade pod and found the oc actually has been extracted. But I can't reproduce it in my end. So I'm not sure if it's a product issue or a prow dedicated issue. The change just tries to work around the issue for now.